rewrite-maven/src/main/java/org/openrewrite/maven/RenamePropertyKey.java

```diff
@@ -50,13 +50,7 @@ public String getDescription() {
 
     @Override
     public TreeVisitor<?, ExecutionContext> getVisitor() {
-        return Preconditions.check(new MavenVisitor<ExecutionContext>() {
-            @Override
-            public Xml visitDocument(Xml.Document document, ExecutionContext executionContext) {
-                // Scanning every tag's value is not an efficient applicable test, so just accept all maven files
-                return SearchResult.found(document);
```

Comment:
Not quite sure what this was for, but it seemed too broad, and not to limit recipe execution by much.